README.md

```diff
@@ -210,13 +210,18 @@ testing purposes.
 This sandbox includes solidus\_auth\_devise and generates with seed and sample
 data already loaded.
 
-* Create the sandbox application (`DB=mysql` or `DB=postgresql` can be specified
-  to override the default sqlite)
+* Create the sandbox application 
 
   ```bash
   bundle exec rake sandbox
   ```
 
+  The sandbox uses SQLite3 by default. If you'd like to use PostgreSQL or MySQL, you can set the `DB` variable for the bash command. (`postgresql` or `mysql`)
```

Comment:
My personal preference would be: 

The sandbox uses SQLite3 by default. You can create a sandbox with PostgreSQL or MySQL by setting the `DB` environment variable.

```
code example
```

Does that seem more readable to you? This is pretty subjective.

Reply:
To be honest, I was struggling to find just the right wording. I've implemented your suggestion.